platforms/documentation/docs/src/docs/userguide/best-practices/best_practices_dependencies.adoc

```diff
@@ -307,3 +287,67 @@ include::sample[dir="snippets/bestPractices/avoidDuplicateDependencies-do/groovy
 === Tags
 
 `<<tags_reference.adoc#tag:dependencies,#dependencies>>`
+
+[[single-gav-string]]
+== Declare Dependencies using a single GAV (`group:artifact:version`) String
+
+When declaring dependencies without a <<version_catalogs.adoc#version-catalog,version catalog>>, prefer using the single GAV string notation `implementation("org.example:library:1.0")`.
+Avoid using the named argument notation unless necessary `implementation(group = "org.example", name = "library", version = "1.0")`.
+
+TIP: The named argument notation may be deprecated in the future.
```

Comment:
```suggestion
TIP: The named argument and map notations may be deprecated in the future.
```

🤔 I think we should show and name the Groovy DSL syntax next to the Kotlin syntax here as well.  Groovy is _not_ using named args, it's using a map, which is then unpacked.  This item should unambiguously recommend against the map as well.

Reply:
Also, https://github.com/gradle/gradle/pull/33699 begins this.